recipes-bsp/alsa-state/alsa-state.bbappend

```diff
@@ -1,4 +1,4 @@
 # Prepend path to override files from upstream recipe
-FILESEXTRAPATHS:prepend:imx-generic-bsp := "${THISDIR}/${PN}/imx-generic-bsp:"
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
What happens when you have other BSP layers in project ? Would it insert itself into those too ?

Reply:
will not impact your case:
- if other BSP layers defines "imx-generic/nxp/mainline-bsp" in MACHINEOVERRIDES will pick related files under the special folder
- if no defined, they can add bbappend and add the same ILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:" to pick up the version in their layers.
alsa-state
├── imx-generic-bsp
│   └── asound.state
├── imx-mainline-bsp
│   └── asound.conf
└── imx-nxp-bsp
    └── asound.conf

Reply:
That’s sounds good then. I am fine with this change